_episodes/00-intro.md

```diff
@@ -1,5 +1,17 @@
 ---
-title: "Introducing high-performance computing"
+title: "Introducing High-Performance Computing"
 questions:
-- Why use a high-performance computing cluster?
+- Why use a large-scale computing system?
 ---
+
+Topics
+
+When it is appropriate to use a cluster
```

Comment:
I think it's important that learners start thinking early on in the workshop about this. I also feel it's important to do away quickly with any misconceptions, fears or unreasonable expectations they may have about HPC.